examples/src/main/kotlin/examples/inference/LeNetModel.kt

```diff
@@ -27,7 +27,7 @@ private val biasInitializer = GlorotUniform(SEED)
 /**
  * Returns classic LeNet-5 model with minor improvements (Sigmoid activation -> ReLU activation, AvgPool layer -> MaxPool layer).
  */
-fun lenet5(): Sequential = Sequential.of(
+fun lenet5(sigmoidActivations:Activations = Activations.Relu): Sequential = Sequential.of(
```

Comment:
Is it a helpful change? Are you going to run it with another Activation function somewhere?

Reply:
btw it's name is missleading, as it is activation, not sigmoid

Reply:
Agreed, just followed the comments for `Sigmoid activation -> ReLU activation` ;-)